core/controller/src/main/scala/whisk/core/loadBalancer/InvokerSupervision.scala

```diff
@@ -230,7 +225,7 @@ class InvokerActor(testAction: String) extends FSM[InvokerState, InvokerInfo] {
     when(UnHealthy, stateTimeout = healthyTimeout) {
         case Event(_: PingMessage, _)                   => stay
         case Event(StateTimeout, _)                     => goto(Offline)
-        case Event(cm: InvocationFinishedMessage, info) => handleCompletionMessage(cm.successful, info.buffer, UnHealthy)
+        case Event(cm: InvocationFinishedMessage, info) => handleCompletionMessage(cm.successful, info.buffer)
```

Comment:
I think we should *always* handle those. Could move the whole method to `onUnhandled` then.